nubis/files/confd/templates/sso-require-macros.conf.tmpl

```diff
@@ -1,24 +1,26 @@
 # Limit Access to Admins
 <Macro RequireOnlyAdmins>
-  <RequireAll>
-    Require claim multifactor:duo
-    <RequireAny>
-{{ if exists "/config/SudoGroups" }}{{ range split (getv "/config/SudoGroups") "," }}      Require claim groups:{{.}}
-{{ end }}{{ end }}
-    </RequireAny>
-  </RequireAll>
+  <RequireAny>
+{{- if exists "/config/SudoGroups" }}{{ range split (getv "/config/SudoGroups") "," }}
+    Require claim groups:{{.}}
+    Require claim https://sso.mozilla.com/claim/groups:{{.}}
```

Comment:
Shouldn't we make the claim URL a configurable option?

Reply:
Not a big deal but I'm ok with these changes

Reply:
Hrm, I believe you do have a very good point there.

Will do and file a separate PR for that.

Reply:
For now I'll r+ this